### PR TITLE
Unfucks smallscreen, removes my stupidity

### LINKED ISSUE #51264: With "Fit Viewport" set to auto and widescreen disabled, changing mobs seems to revert back to wide screen.
Round ID: [137944](https://scrubby.melonmesa.com/round/137944)

Reproduction: Have the following settings:

Fit Viewport: Auto
Widescreen: Disabled (15x15)

Observe. You'll probably be put into widescreen. If not, then spawn yourself as a human from there. You will change to widescreen, toggling your widescreen preference to enabled/disabled again will fix it. Now set Fit Viewport to Manual and make sure you're not in widescreen. Changing mobs has no effect.

### LINKED ISSUE #51265: view range fails to increase when using the whiteship nav console
Issue reported from  Round ID: [137945](https://scrubby.melonmesa.com/round/137945) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1524

### LINKED ISSUE #51273: Preference for Pixel scaling fails to save.
it defaults to stretch to fit every round

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

So uh #51260 fixed but also broke part of #51208, namely the part in login().
I didn't see it in testing because it was a hotfix, but as it currently stands a players's view will reset quite often.

I've changed it to use getScreenSize along with prefs.widescreen,  which should work.
I've made absolutely sure that isZooming() is doing what it should, perhaps more then I should have
Removed an instance of setDefault(), I decided it was dumb
I never actually set up pixel_size and scaling_method to save, that is fixed now.
And I tweaked a ghost verb to work properly with my changes to rescale_view()

## Why It's Good For The Game

My stupidity should not effect players visuals.
Fixes #51264
FIxes #51265
Fixes #51273
## Changelog
:cl: LemonInTheDark
fix: Fixed smallscreen resizing on restart or when you die.
fix: Pixel size and scaling method will properly save now.
/:cl:

### CODE REVIEW

**Comment by BeeSting12** [comment]
Does this fix #51264?

**Comment by LemonInTheDark** [comment]
Should do, testing with the details you provided currently

**Comment by LemonInTheDark** [comment]
Yep it fixes it

**Comment by LemonInTheDark** [comment]
Fixed the nav console too

**Comment by BeeSting12** [comment]
what about the pixel scaling pref not saving? it always does stretch to fit

**Comment by LemonInTheDark** [comment]
Yep, got that too

File: code/game/machinery/computer/camera_advanced.dm [hunk 1]
```diff
@@ -11,7 +11,8 @@
 	var/datum/action/innate/camera_off/off_action = new
 	var/datum/action/innate/camera_jump/jump_action = new
 	var/list/actions = list()
-
+	///Should we supress any view changes?
+	var/should_supress = TRUE
```
Comment (Rohesie) [inline code/game/machinery/computer/camera_advanced.dm:15]:
```suggestion
	var/should_supress_view_changes = TRUE
```
Informative var name.

Comment (LemonInTheDark) [reply code/game/machinery/computer/camera_advanced.dm]: gottcha

**Pull request merged**